app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More about teachers' salaries and benefits</h2>
+
+    <div class="inset">
+      <p>
+        Your teaching salary will depend on your role in the school and qualifications.
+        There are lots of opportunities to increase your salary during your career:
+      </p>
+
+      <ul>
+        <li>teachers who make a sustained contribution to their school could expect to earn £40k to £43k or more in London</li>
+        <li>headteachers are usually the highest paid people in a school and can expect to earn around £50k to £123k depending on the school and location</li>
```

Comment:
```suggestion
        <li>headteachers are usually the highest paid people in a school and can expect to earn around £50k to £131k depending on the school and location</li>
```
@philspray If I'm reading our salaries page correctly, the top pay is more than 123k in London?